src/visitors/symtab_visitor_helper.hpp

```diff
@@ -149,6 +149,9 @@ void SymtabVisitor::setup_symbol(ast::Node* node, NmodlType property) {
 
 void SymtabVisitor::add_model_symbol_with_property(ast::Node* node, NmodlType property) {
     auto token = node->get_token();
+    if (!token) { // token can be nullptr
+        return;
+    }
```

Comment:
@alkino : I believe in this case we need to add `node` to symbol table but with "default" `ModToken` object (i.e. token which will not have location information associated with it).

Otherwise, here we are just skipping adding symbol in the symbol table which means we will not find that symbol and that might be problematic somewhere else.